### PR TITLE
Add chain registry for acrechain

### PR DESCRIPTION
**Pull request by gophereth**

### CODE REVIEW

File: acrechain/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Acrechain",
+      "denom_units": [
+        {
+          "denom": "aacre",
+          "exponent": 0
+        },
+        {
+          "denom": "ACRE",
```
Comment (JeremyParish69) [inline acrechain/assetlist.json:13]: can you make this lowercase please so it matches the 'display' denom.

Comment (gophereth) [reply acrechain/assetlist.json:13]: Okay

File: acrechain/chain.json [hunk 2]
```diff
@@ -0,0 +1,207 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "acrechain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://arable.finance/",
+  "pretty_name": "Acrechain",
+  "chain_id": "acre_9052-1",
+  "bech32_prefix": "acre",
```
Comment (JeremyParish69) [inline acrechain/chain.json:9]: and please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: acrechain/assetlist.json (hunk 1 above)
Comment (gophereth) [reply acrechain/assetlist.json]: Done

File: acrechain/chain.json (hunk 2 above)
Comment (gophereth) [reply acrechain/chain.json:9]: Raised the PR and waiting for slip-0173
https://github.com/satoshilabs/slips/pull/1459

**Pull request merged**